### DISCUSSION TITLE
Ability to show surrounding lines to a search

### DISCUSSION THREAD

**Discussion by devilmanzato** [variar/klogg#426] [Ideas]
Hi!
I believe that we could benefit from an option where we could select search to show us the surrounding lines to the search.
For example, when searching for a crash, being able to enable a flag to show two lines before and two lines after could be helpful , especially when trying to identify repetitive patterns.

Perhaps something like what exist for Grep:
-A num
--after-context=num

    Print num lines of trailing context after matching lines.
-B num
--before-context=num

    Print num lines of leading context before matching lines.
-C num
-num
--context=num

    Print num lines of leading and trailing output context.

**Comment by variar** [comment]
Hi, thanks for suggestion. That is indeed a useful feature. There is a featue request for it here #266
